TKeazirian.HTTPServer/Router/Route.cs

```diff
@@ -0,0 +1,22 @@
+namespace TKeazirian.HTTPServer.Router;
+
+using Handler;
+
+public class Route
```

Comment:
A Route as an object. I wonder if this is useful, or adding a level on unneeded complexity.

Reply:
To me, it seemed like the best way to hold the data and allows for me to have the path, methods, and handler all in one object.